styles/namemc/catppuccin.user.css

```diff
@@ -0,0 +1,765 @@
+@description Soothing pastel theme for NameMC
+@author Catppuccin
+@license MIT
+
+@preprocessor less
+@var select lightFlavor "Light Flavor" ["latte:Latte*", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha"]
+@var select darkFlavor "Dark Flavor" ["latte:Latte", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha*"]
+@var select accentColor "Accent" ["rosewater:Rosewater", "flamingo:Flamingo", "pink:Pink", "mauve:Mauve*", "red:Red", "maroon:Maroon", "peach:Peach", "yellow:Yellow", "green:Green", "teal:Teal", "blue:Blue", "sapphire:Sapphire", "sky:Sky", "lavender:Lavender", "subtext0:Gray"]
+==/UserStyle== */
+
+@-moz-document domain('namemc.com') {
+  :root[data-bs-theme="dark"] {
+    #catppuccin(@darkFlavor, @accentColor);
+
+    --bs-body-color: @text;
+    --bs-body-color-rgb: @text;
+    --bs-body-bg: @mantle;
+    --bs-body-bg-rgb: @mantle;
+    --bs-emphasis-color: @text;
+    --bs-emphasis-color-rgb: @text;
+    --bs-secondary-color: @subtext1;
+    --bs-secondary-color-rgb: @subtext1;
+    --bs-secondary-bg: @surface0;
+    --bs-secondary-bg-rgb: @surface0;
+    --bs-tertiary-color: @subtext0;
+    --bs-tertiary-color-rgb: @subtext0;
+    --bs-tertiary-bg: @base;
+    --bs-tertiary-bg-rgb: @base;
+    --bs-primary-text-emphasis: @sapphire;
+    --bs-secondary-text-emphasis: @subtext0;
+    --bs-success-text-emphasis: @accent-color;
+    --bs-info-text-emphasis: @sky;
+    --bs-warning-text-emphasis: @yellow;
+    --bs-danger-text-emphasis: @maroon;
+    --bs-light-text-emphasis: @text;
+    --bs-dark-text-emphasis: @text;
+    --bs-primary-bg-subtle: @crust;
+    --bs-secondary-bg-subtle: @crust;
+    --bs-success-bg-subtle: @crust;
+    --bs-info-bg-subtle: @crust;
+    --bs-warning-bg-subtle: @surface0;
+    --bs-danger-bg-subtle: @base;
+    --bs-light-bg-subtle: @surface0;
+    --bs-dark-bg-subtle: @mantle;
+    --bs-primary-border-subtle: @mantle;
+    --bs-secondary-border-subtle: @surface1;
+    --bs-success-border-subtle: @accent-color;
+    --bs-info-border-subtle: @teal;
+    --bs-warning-border-subtle: @peach;
+    --bs-danger-border-subtle: @red;
+    --bs-light-border-subtle: @surface1;
+    --bs-dark-border-subtle: @surface0;
+    --bs-heading-color: inherit;
+    --bs-link-color: @accent-color;
+    --bs-link-hover-color: @text;
+    --bs-link-color-rgb: @accent-color;
+    --bs-link-hover-color-rgb: @text;
+    --bs-code-color: @accent-color;
+    --bs-border-color: @surface1;
+    --bs-border-color-translucent: @surface1;
+    --bs-form-valid-color: @green;
+    --bs-form-valid-border-color: @green;
+    --bs-form-invalid-color: @red;
+    --bs-form-invalid-border-color: @red;
+
+    .mc-black {
+      color: @crust !important;
+    }
+    .mc-dark-gray {
+      color: @overlay0 !important;
+    }
+    .mc-gray,
+    .mc-reset {
+      color: @subtext0 !important;
+    }
+    .mc-white {
+      color: @text !important;
+    }
```

Comment:
you can deduplicate this by moving it to the `#catppuccin()` mixin with no additional selectors